Assets/Live2D/Cubism/Rendering/CubismRenderController.cs

```diff
@@ -838,7 +844,7 @@ private void OnDynamicDrawableData(CubismModel sender, CubismDynamicDrawableData
 
 
                 // Update render order if last SwapInfo flags is true.
-                renderers[i].UpdateRenderOrder();
+                IsRenderOrderChanged |= renderers[i].UpdateRenderOrder();
```

Comment:
```suggestion
                IsRenderOrderChanged |= data[i].IsRenderOrderDirty;
```
You should be able to substitute the use of `data[i].IsRenderOrderDirty` without having to modify `UpdateRenderOrder()`. Please try the suggested solution.